server/controllers/index.js

```diff
@@ -5,7 +5,7 @@ const { login } = require('./login')
 const { getGuides } = require('./guides')
 const { isAuthenticated } = require('../middleware/auth')
 
-router.get('/places', isAuthenticated, getPlaces)
+router.get('/places', getPlaces, isAuthenticated)
```

Comment:
isAuthenticated is middleware that needs to go before getPlaces to protect the route. Middleware goes it order so should hit authentication first. Does this make sense?

Reply:
i did it because until now we didn't check if the user is logged in or not, because this will be when the user clicked on the add review button, and Saja is working on this component now.

Reply:
Okay I would delete isAuthenticated from here then until it works :)